src/main/java/edu/tamu/app/controller/ButtonManagementController.java

```diff
@@ -37,7 +37,7 @@ public ApiResponse create(@RequestBody PersistedButton button) {
     @RequestMapping("/{id}")
     @PreAuthorize("hasRole('ADMIN')")
     public ApiResponse getButton(@PathVariable Long id) {
-        return new ApiResponse(SUCCESS, persistedButtonRepo.findOne(id));
+        return new ApiResponse(SUCCESS, persistedButtonRepo.findById(id));
```

Comment:
`findById` is returning an Optional. You will have to use another method or call .`get()` on the Optional.